package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "business-edit-ui",
-  "version": "3.0.0",
-  "lockfileVersion": 2,
+  "version": "3.0.2",
+  "lockfileVersion": 1,
```

Comment:
You did not update your node version before rebuilding this file. This will cause errors in our build server.